src/includes/txs/wallet_pool.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "constants.h"
+#include "uthash.h"
+#include "utxo_pool.h"
+
+typedef struct {
+  mbedtls_ecdsa_context *key_pair;
+  unsigned long amt;
+  int spent;  // True when included in a tx but not validated in a block
```

Comment:
how do we seperate in a tx, in a block, in neither?

Reply:
Not sure I follow.

Reply:
Asking what are the values we expect spent to be, if we are including it is in the blockchain, vs in a transaction made by the wallet but not yet confirmed, and when it is just chilling

Reply:
I was imagining 0 = never spent, 1 = spent but not confirmed, and included in a block means removed from the wallet pool